### PR TITLE
set confluence.home to homedir if data_dir is empty

### PR DESCRIPTION
**Pull request by kpankonen**
for #111

### CODE REVIEW

**Comment by kpankonen** [comment]
after the 2.3.0 merge, the tests are all passing.  do i need to add anything else?

File: templates/confluence-init.properties.erb [hunk 1]
```diff
@@ -32,7 +32,7 @@
 # specify your directory below (don't forget to remove the '#' in front)
 
 # confluence.home=c:/confluence/data
-<% if scope.lookupvar('confluence::data_dir') -%>
+<% if ! scope.lookupvar('confluence::data_dir').empty?  -%>
```
Comment (binford2k) [inline templates/confluence-init.properties.erb]: A better fix would likely be to set the default to `undef` to begin with. See https://github.com/voxpupuli/puppet-confluence/blob/dc96ab3f0c0cb26bbda219ef90099f472f92faa1/manifests/init.pp#L20

**Comment by kpankonen** [comment]
@binford2k updated default to be undef and removed the empty check

**Comment by kpankonen** [comment]
@bastelfreak sorry to bother you, but I was wondering if you could help get this merged?

**Comment by oranenj** [comment]
@kpankonen this actually looks like a bugfix, and the tests seem legit, so I'll just merge this. Thanks

**Pull request merged**